.github/workflows/dotnet.yml

```diff
@@ -33,11 +33,11 @@ jobs:
           9.0.x
 
     - name: Restore dependencies
-      run: dotnet restore
+      run: dotnet restore ./NCronJob.sln
```

Comment:
Let's delete the SLNX for now. It is still not stable

Reply:
Fixed